reference/api/search.md

```diff
@@ -67,19 +67,20 @@ Query terms enclosed in double quotes are treated as [phrase searches](#query-q)
 
 ### Response
 
-| Name                     | Type             | Description                                     |
-| :----------------------- | :--------------- | :---------------------------------------------- |
-| **`hits`**               | Array of objects | Results of the query                            |
-| **`offset`**             | Number           | Number of documents skipped                     |
-| **`limit`**              | Number           | Number of documents to take                     |
-| **`estimatedTotalHits`** | Number           | Estimated total number of matches               |
-| **`totalHits`**          | Number           | Exhaustive total number of matches              |
-| **`totalPages`**         | Number           | Exhaustive total number of search result pages  |
-| **`hitsPerPage`**        | Number           | Number of results on each page                  |
-| **`page`**               | Number           | Current search results page                     |
-| **`facetDistribution`**  | Object           | **[Distribution of the given facets](#facets)** |
-| **`processingTimeMs`**   | Number           | Processing time of the query                    |
-| **`query`**              | String           | Query originating the response                  |
+| Name                     | Type             | Description                                                     |
+| :----------------------- | :--------------- | :-------------------------------------------------------------- |
+| **`hits`**               | Array of objects | Results of the query                                            |
+| **`offset`**             | Number           | Number of documents skipped                                     |
+| **`limit`**              | Number           | Number of documents to take                                     |
+| **`estimatedTotalHits`** | Number           | Estimated total number of matches                               |
+| **`totalHits`**          | Number           | Exhaustive total number of matches                              |
+| **`totalPages`**         | Number           | Exhaustive total number of search result pages                  |
+| **`hitsPerPage`**        | Number           | Number of results on each page                                  |
+| **`page`**               | Number           | Current search results page                                     |
+| **`facetDistribution`**  | Object           | **[Distribution of the given facets](#facetdistribution)**      |
+| **`processingTimeMs`**   | Number           | Processing time of the query                                    |
+| **`query`**              | String           | Query originating the response                                  |
+| **`facetStats`**         | Object           | [The the numeric `min` and `max` values per facet](#facetstats) |
```

Comment:
I would have put `facetStats` just below `facetDistribution` for the reading ease

Reply:
Good idea!